tests/examples/examples_test.go

```diff
@@ -157,7 +169,7 @@ func TestExamples(t *testing.T) {
 				t *testing.T, stackInfo integration.RuntimeValidationStackInfo,
 			) {
 				assert.NotNil(t, stackInfo.Deployment)
-				assert.Equal(t, 9, len(stackInfo.Deployment.Resources))
+				assert.Equal(t, 15, len(stackInfo.Deployment.Resources))
```

Comment:
We can punt this off to another PR, but i've noticed this hard-coding of numbers to expect in assertions before and IIRC we've talked about it, but I was wondering if we can move away to a computed value instead.

Reply:
Yeah, I'll punt for now. It's something I have in mind for test cleanup.